### ISSUE TITLE
[Accessibility- Tab order C 12.8] Notes- Ensure consistent tab order

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3053]
**Environment:** Tribe

**Current behavior:**

In Notes page, we don't have a consistent tabbing path through content and functionality:

- Tabbing does not allow access to the functionalities of modal windows (drawer) by clicking directly on the buttons that allow opening it: The tree Notes
- Tabbing does not allow access to the menu of actions(copy link, Print to PDF....)on a Note page
- Tabbing does not allow the focus resumption on the Note page: When enabling the Open/Edit page, a new page is opened and the user can't return to the Note page by tabbing

**Expected behavior:**

- Tabbing shouldn't reach visually hidden elements.

- When activating a button that opens a modal window, the focus should be moved to the first element of the window, and when closing it, the focus should return to the activation button

**Comment by mbensalem** [comment]
Not relevant any more
